readme.txt

```diff
@@ -53,6 +53,27 @@ This should be a path (i.e. `/test`) or a URL (i.e. `http://example.com/wp/test`
 
 == Changelog ==
```

Comment:
This is getting quite long, as [files over 10KB can cause problems](https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/#file-size) I recommend removing all the entries for 1.x versions and replacing them with a link to the changelog on GitHub.

The file is currently 28K.